src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -65,7 +82,7 @@ void Terminal::Create(COORD viewportSize, SHORT scrollbackLines, IRenderTarget&
 {
     _mutableViewport = Viewport::FromDimensions({ 0,0 }, viewportSize);
     _scrollbackLines = scrollbackLines;
-    COORD bufferSize { viewportSize.X, viewportSize.Y + scrollbackLines };
+    COORD bufferSize { viewportSize.X, _ClampToShortMax(viewportSize.Y + scrollbackLines, 1) };
     TextAttribute attr{};
     UINT cursorSize = 12;

```

Comment:
I KNOW this wasn't something you changed. But can you hit it while you're nearby?

```suggestion
    const UINT cursorSize = 12;
```